assignments/this.js

```diff
@@ -1,26 +1,67 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Implicit Binding is when all things left of the dot in dot notation is invoked.
+* 2. Explicit Binding is when .call(), .bind(), and .apply() are invoked.
+* 3. Global or Default Binding is when neither implicit or explicit binding is used, and the "this" becomes invoked.
+* 4. New Binding is when we use new to create a keyword.
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
 
```

Comment:
All of these examples are copied directly from the TK. Please complete this portion of the assignment with your own examples.